egress/egressonlytracking.yaml

```diff
@@ -47,21 +47,26 @@ components:
       type: object
       required:
       - name
-      - offset
+      - rx_offset
       properties:
         name:
           description: |-
-            The Name used to identify the metrics associated with the values applicable
-            for configured offset and length inside corresponding header field.
+            The name used to identify the metric tracked at configured Rx offset and of configured
+            length. Tx offset configuration is optional.
           type: string
           pattern: ^[\sa-zA-Z0-9-_()><\[\]]+$
           x-field-uid: 1
-        offset:
+        rx_offset:
           description: |-
-            Offset in bits relative to start of the packet.
+            Rx Offset in bits relative to start of the packet.
           type: integer
           format: uint32
           x-field-uid: 2
+        tx_offset:
+          description: |-
+            Tx Offset in bits relative to start of the packet.
```

Comment:
Maybe this is the right place to include the optionality and when it is needed. Also description should move to custom probably. 
i.e. Container for  optional tx offset settings . This is an optional field and is relevant only when optional Tx statistics in egress only tracking are supported and required while fetching egress only stats by enabling "tx_metric" in get_metrics/egress_only_tracking/tagged_metric/metric_names .